.github/workflows/deploy-apis-to-production.yml

```diff
@@ -350,9 +355,9 @@ jobs:
 
       - name: Deploy to k8s cluster
         run: |
-          export MEASUREMENTS_CONNECT_IMAGE=${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/measurements-connect:${{ needs.image-tag.outputs.build_id }}
-          export STAGE_KAFKA_CLUSTER=${{ secrets.STAGE_KAFKA_CLUSTER }}
-          cat k8s/kafka/connectors/measurements-connect.yaml | sed "s/{{MEASUREMENTS_CONNECT_IMAGE}}/$MEASUREMENTS_CONNECT_IMAGE/g" | sed "s/{{KAFKA_CLUSTER}}/$STAGE_KAFKA_CLUSTER/g" | kubectl apply -n message-broker -f-
+          export MEASUREMENTS_CONNECT_IMAGE=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/measurements-connect:${{ needs.image-tag.outputs.build_id }}
+          export PROD_KAFKA_CLUSTER=${{ secrets.PROD_KAFKA_CLUSTER }}
+          cat k8s/kafka/connectors/measurements-connect.yaml | sed "s/{{MEASUREMENTS_CONNECT_IMAGE}}/$MEASUREMENTS_CONNECT_IMAGE/g" | sed "s/{{KAFKA_CLUSTER}}/$PROD_KAFKA_CLUSTER/g" | kubectl apply -n message-broker -f-
```

Comment:
[use the kafka PROD connector in production](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/ca58771bfead5adf55788933aec52ae5759582c6)